src/journal-remote/journal-gatewayd.c

```diff
@@ -1040,7 +1040,7 @@ int main(int argc, char *argv[]) {
                 int flags =
                         MHD_USE_DEBUG |
                         MHD_USE_DUAL_STACK |
-                        MHD_USE_PIPE_FOR_SHUTDOWN |
+                        MHD_USE_ITC |
```

Comment:
Thanks for checking that out. I wanted to put the version number in the commit message, but I was too lazy yesterday ;)

My commit adds backwards-compat mapping in microhttp-util.h. It should compile fine against old µhttpd versions.